semeio/jobs/ots/ots_config.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+import configsuite
+import datetime
+
+from configsuite import MetaKeys as MK
+from configsuite import types
+from copy import deepcopy
+
+
+@configsuite.validator_msg("List needs to be of size 2")
+def _is_length_equal_2(value):
+    return len(value) == 2
+
+
+@configsuite.validator_msg("Vintages must contain at least an entry!")
+def _min_length(value):
+    return len(value) > 1
+
+
+@configsuite.transformation_msg("Converting list of strings to list of dates")
+def _str2dates(value):
+    value = deepcopy(value)
+    dates = []
+    for x in value:
+        if isinstance(x, str):
+            dates.append(datetime.datetime.strptime(x, "%Y-%m-%d").date())
+        else:
+            dates.append(x)
+    return dates
+
+
+@configsuite.validator_msg("OTS vintage date must be present in .UNRST file!")
+def _vintage_present_in_rst(elem, context):
+    return elem in context
+
+
+def build_schema():
+    return {
+        MK.Type: types.NamedDict,
+        MK.Description: "Overburden time shift job parameters",
+        MK.Content: {
+            "seabed": {
+                MK.Type: types.Number,
+                MK.Description: "The depth of the seabead in meters.",
+            },
+            "rfactor": {
+                MK.Type: types.Number,
+                MK.Description: "Scales the surface displacement between"
+                " base_survey and monitor_survey, eg. 20.",
+            },
+            "above": {
+                MK.Type: types.Number,
+                MK.Description: "Distance in meters above the reservoir"
+                " where shift is calculated. The distance from the "
+                "shallowest cell, eg. 100",
+            },
+            "convention": {
+                MK.Type: types.Number,
+                MK.Description: "Positive or negative shift can be either 1 or -1,"
```

Comment:
If it can be only 1 or -1, I think we should add an element validator here.

Reply:
Sounds like a good idea!